<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAluguel</class>
 <widget class="QDialog" name="FrmAluguel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aluguel de Veículos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Imagens/btnAlugaVeiculo.png</normaloff>../../Imagens/btnAlugaVeiculo.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data do Aluguel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtDataAluguel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtPrazo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prazo de Entrega</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data da Devolução</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtDataDev">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtValor">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor Aluguel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtMulta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor Multa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtkmEntrada">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Km de Entrada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Km de Saída</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtKmSaida">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Selecione o Cliente</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Pesquisa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPesqCliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>Código</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CPF</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="EdtPesqCliente">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPesqCliente">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Imagens/lupa.png</normaloff>../../Imagens/lupa.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="gridCliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CPF</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Endereço</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Telefone</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Selecione o Veículo</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Pesquisa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPesqVeic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>Código</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marca</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Modelo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Disponível</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alugado</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="EdtPesqVeiculo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPesqVeic">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Imagens/lupa.png</normaloff>../../Imagens/lupa.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="gridVeiculo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modelo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marca</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ano</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Placa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Alugado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Batido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Km Atual</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor Diária</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descrição</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSalvar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Imagens/save.png</normaloff>../../Imagens/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
